charts/kubewarden-controller/tests/webhooks_existing_certifcates_test.yaml

```diff
@@ -0,0 +1,71 @@
+suite: existing certificates are not overwritten
+templates:
+  - webhooks.yaml
+release:
+  namespace: "kubewarden"
+kubernetesProvider:
+  scheme:
+    "v1/Secret":
+      gvr:
+        version: "v1"
+        resource: "secrets"
+      namespaced: true
+  objects:
+    - kind: Secret
+      apiVersion: v1
+      metadata:
+        name: kubewarden-ca
+        namespace: kubewarden
+      data:
+        ca.crt: "Y2EuY3J0" # "ca.crt" in base64
+        ca.key: "Y2Eua2V5" # "ca.key" in base64
+        old-ca.crt: "b2xkLWNhLmNydA==" # the string "old-ca.crt" in base64
+    - kind: Secret
+      apiVersion: v1
+      metadata:
+        name: kubewarden-webhook-server-cert
+        namespace: kubewarden
+      data:
+        tls.crt: "dGxzLmNydA==" # "tls.crt" in base64
+        tls.key: "dGxzLmtleQ==" # "tls.key" in base64
+tests:
+  - it: "should generate a CA certificate and store it in the kubewarden-ca secret"
```

Comment:
I think the text is misleading, because it's not actually generating anything since the secrets are already there.